src/test/java/org/sagebionetworks/bridge/workerPlatform/bridge/BridgeHelperTest.java

```diff
@@ -624,6 +631,49 @@ public void redriveUpload_timeout() throws Exception {
         verify(mockWorkerApi, times(3)).getUploadById(UPLOAD_ID);
     }
 
+    @Test
+    public void testGetAccountSummariesForApp() throws IOException {
+        // mock SDK search account summaries call
+        AccountSummaryList accountSummaryList = mock(AccountSummaryList.class);
+        when(accountSummaryList.getItems()).thenReturn(ImmutableList.of(new AccountSummary()));
+        Response<AccountSummaryList> response = Response.success(accountSummaryList);
+
+        Call<AccountSummaryList> mockCall = mock(Call.class);
+        when(mockCall.execute()).thenReturn(response);
+
+        when(mockWorkerApi.searchAccountSummariesForApp(eq(APP_ID), any(AccountSummarySearch.class))).thenReturn(mockCall);
```

Comment:
We should use a Captor to verify the params set in the AccountSummarySearch. We should verify the pageSize, offset, orgId, and studyId.

Also, we'll need 2 test cases, one for orgId and one for studyId.